app/views/job_templates/run_openscap_remediation_-_ansible_default.erb

```diff
@@ -22,6 +22,14 @@ require:
 ---
 - hosts: all
   tasks:
+<%= indent(4) { snippet('check_bootc_status') } %>
+    - name: Enable bootc overlay
+      shell:
+        cmd: 'bootc usr-overlay'
+      register: out
+      ignore_errors: true
+      when: is_bootc_host
+      - debug: var=out
```

Comment:
The indentation looks off
```suggestion
    - debug: var=out
```

Reply:
Thanks, I haven't created a setup to actually test this myself